Studio-Lovelies/GG-JointJustice-Unity
# Adds `&LOCK_MOVE_CHOICE` and `&LOCK_TALK_CHOICE` actions

## Summary
Adds `&LOCK_MOVE_CHOICE` and `&LOCK_TALK_CHOICE` actions.

## Testing instructions
1. Open the Game scene
2. Set the `GameStarter` to `InvestigationUI`
3. Select talk option 2 and notice talk option 3 being available
4. Select talk option 3 and notice talk option 2 missing afterwards
5. Select talk option 1 and notice talk option 2 reappearing

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - #494
